Dockerfile

```diff
@@ -14,30 +14,22 @@ RUN yum -y install epel-release &&\
     yum -y install findutils &&\
     yum clean all
 
-RUN chgrp -R root ${ROOT_GROUP_DIRS} &&\
-    chmod -R g+rwx ${ROOT_GROUP_DIRS}
 
-COPY . /tmp
-
-WORKDIR /tmp
+COPY requirements.txt /requirements.txt
+RUN chgrp -R root ${ROOT_GROUP_DIRS} && \
+    chmod -R g+rwx ${ROOT_GROUP_DIRS} && \
+    pip3 install --no-cache-dir -r /requirements.txt
 
+COPY nginx.conf /etc/nginx
+EXPOSE 8000
+CMD [ "/usr/sbin/nginx" ]
 
-RUN git clone --no-checkout https://github.com/CSCfi/csc-user-guide git_folder && \
-    if [ -d ".git" ]; then rm -r .git; fi && \
-    mv git_folder/.git . && \
-    rm -r git_folder && \
-    git reset HEAD --hard && \
```

Comment:
AFAIK this is still needed to generate the modification dates in the pages.

![Screenshot from 2024-03-25 14-27-51](https://github.com/CSCfi/csc-user-guide/assets/2276349/035be79c-3d49-4175-a216-43c37a0edfb0)

Reply:
Which is not present in the build done by Rahti BC. Keep in mind that the preview and the production are very different processes.

Reply:
🤔 the git log says that the page was modified March 7 2023. 

The Last update  seems to be the same on the feature branch as well as master:
https://csc-guide-preview.rahtiapp.fi/origin/testing-speedup/cloud/
https://docs.csc.fi/cloud/

There is of course something that I still might be missing.

I'm also wondering that IF it is still needed maybe it would be possible to use `ADD --keep-git-dir=true` instead of `COPY . /tmp`

Reply:
In a Build config OpenShift clones the code automatically but removes the `.git` directory before starting to run `Dockerfile`. I can prepare a demo example later today/tomorrow.